### PR TITLE
OOP school library: refactor your code

### PR DESCRIPTION
**Pull request by RashidMenhas**
## In this milestone, we implemented the following:

- [ ]  Paid special attention to the "Single-responsibility Principle".

- [ ] Made sure that the role of main.rb is only to start the code and ask the user for the initial choice of options (or to quit).

- [ ] Extracted some pieces of logic into a separate class(es).

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -111,8 +111,41 @@ def list_rental_by_id
...
+
+  def options(*)
+    loop do
+      display_all
+      option = gets.chomp.to_i
+      case option
+      when 1
+        list_of_people
+      when 2
+        list_of_books
```
Comment (Abenezer-Tilahun) [inline app.rb:137]: - [ optional ] Kindly I recommend you to fix the selection issues with List all books and List all people., when  I select the 1 it list list out peoples same for when I select 2 it list list out books as show the below screenshot.

Capture

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @RashidMenhas @demix007 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Git-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for Implemented OOP school library: refactor your code.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by demix007** [comment]
## Hello @Abenezer-Tilahun :wave,

Thank you for taking out time to review and approve this PR. You are awesome 💯

File: app.rb (hunk 1 above)
Comment (demix007) [reply app.rb:137]: Thank you for this change that you recommended. I have gone ahead to Implement the selection issue.

**Pull request merged**